### ISSUE TITLE
Builds for older ffmpeg versions

### ISSUE THREAD

**Issue by nevaran** [m-ab-s/media-autobuild_suite#1988]
Is it possible to have builds of an older ffmpeg build? [This](https://github.com/BtbN/FFmpeg-Builds/releases/tag/autobuild-2020-09-30-23-26) one in particular.

The reason being the project I am developing has a dependency package that cannot run well (or at all with latest builds) of ffmpeg, and since I am just using the gif and webm support on it, theres quite a lot of unnecessary library size.

**Comment by 1480c1** [comment]
In theory, it's possible to do so using https://github.com/m-ab-s/media-autobuild_suite#custom-patches, but this is of course not supported by the suite and is a case by case of if we will respond to such requests since it's somewhat annoying if we tried to tailor a script for everybody that requests a specific version.

Note, they are not guaranteed to always work, so if they break, you will probably need to try to fix it yourself

**Comment by nevaran** [comment]
I see. So ill just have to stay with the full library then

**Comment by Biswa96** [comment]
Some GNU/Linux distributions can provide older versions of mingw-w64 ffmpeg. e.g. For ArchLinux/AUR, there is a whole pacman repository https://github.com/Martchus/PKGBUILDs

**Comment by nevaran** [comment]
Ah alright, thanks for the info!

**Comment by hotenov** [comment]
@1480c1 Hello!

Can you explain in more details how to write this `/build/ffmpeg_extra.sh` file for existing local folder with cloned source code of ffmpeg previous version?

I looked at the example for  `/build/ffmpeg_extra.sh` in README, but there is example with patches. I have no patches, I have a full cloned repo, but for certain tag. How to prevent script for downloading updates from remote latest 'master' version?

I also tried to add `ffmpegPath=../myrepos/ffmpeg` line to `build\media-autobuild_suite.ini` but script `media-autobuild_suite.bat` ignores this settings and take `--ffmpegPath=https://git.ffmpeg.org/ffmpeg.git` in any case _(according to `last_run` file)_

Also can you explain how to set relative path to local cloned repo? Where script will search the local repo for path `../myrepos/ffmpeg` in `mabs/myrepos/ffmpeg` OR in `mabs/build/myrepos/ffmpeg` OR _<your_right_answer>_?

To sum up, How to tell script "Configure, make and install ONLY ffmpeg from this local folder. Do NOT UPDATE git repo, just take it as-is" ?
I understand that this is a bad idea, but my old video card converts MP4 only with older ffmpeg `n4.4.2` and I need only old version of built-in  `libavcodec` library, because they broke something in `h264_nvenc` since that version :(

OR maybe you can provide an instruction how to build the older ffmpeg version, having functions and your .sh scripts as reference? Which line(s) I should change to forbid script to update ffmpeg repo _(I can manually replace `ffmpeg-git` with my cloned old repo :), I hope)_

Thanks in advance for any help! ;)
